packages/automator/index.tsx

```diff
@@ -175,8 +207,21 @@ const singleProcess = async (
     // returning metadata for aggregation
     return { metadata, state: optimizedState };
   } else {
-    fs.writeFileSync(out, canvas);
-    console.log(chalk.green(`The diagram has been saved as ${out}`));
+    if (staged) {
+      // write multiple svg files out
+      const writeFileOut = (canvasData: any, index: number) => {
+        let filename = out.slice(0, out.indexOf("svg") - 1);
```

Comment:
Perhaps use [`path.join`](https://nodejs.org/api/path.html#path_path_join_paths) and templated strings for readability